bindings/node.js/binding.gyp

```diff
@@ -0,0 +1,51 @@
+{
+  'targets': [
+    {
+      'target_name': 'blst-node-binding',
```

Comment:
Shouldn't 'target_name' be just 'blst'? My impression is that this how the shared library will be named, and it's also the name it will be referred by in the 'require' statement in the script...